polyfill/lib/ecmascript.mjs

```diff
@@ -5726,6 +5726,21 @@ export function ASCIILowercase(str) {
   ]);
 }
 
+export function ValueOfThrows(constructorName) {
+  const compareCode =
+    constructorName === 'PlainMonthDay'
+      ? 'Temporal.PlainDate.compare(obj1.toPlainDate(year), obj2.toPlainDate(year))'
+      : `Temporal.${constructorName}.compare(obj1, obj2)`;
+
+  throw new TypeError(
+    'Do not use built-in arithmetic operators with Temporal objects. ' +
+      `When comparing, use ${compareCode}, not obj1 > obj2. ` +
+      "When coercing to strings, use obj.toString() or String(obj), not +obj nor '' + obj. " +
+      'When concatenating with strings, use str.concat(obj) or `${str}${obj}`, not str + obj. ' +
+      'In React, call obj.toString() before rendering a Temporal object.'
+  );
```

Comment:
That's a lot of quote variance. For sanity, I think maybe just ``` ` ``` with post-processing would be better.
```suggestion
  const rawMessage = `
    Do not use built-in arithmetic operators with Temporal objects.
    When comparing, use ${compareCode} rather than obj1 > obj2.
    When coercing to string, use obj.toString() or String(obj) rather than '' + obj.
    When concatenating to a string, use str.concat(obj) or \`\${str}\${obj}\` rather than str + obj.
    In React, call obj.toString() before rendering a Temporal object.
  `;
  throw new TypeError(rawMessage.trim().replace(/\s+/g, ' '));
```

Reply:
The quote variance comes from prettier; it's automatic. And anyone who's seen a lot of prettier code shouldn't be surprised by this. Also, I'm not convinced that this change would add much extra readability. So I'm inclined to merge as-is. Holler if you strongly disagree!

Reply:
No, not a big deal.